### ISSUE TITLE
Extra tabs need to be overhauled

### ISSUE THREAD

**Issue by Greatexperiment** [deliciousmods/1956_reboot#3]
The expanded equipment tab, and expanded research, are both interesting and needed to some extent. However, they should be overhauled and either consolidated or heavily revised.

Originally the first tab was meant to be extra equipment, and the second tab was to be a new doctrine tree based partially on ideology. 

As per usual, please place all ideas and related thoughts here - and then we'll break it down into tasks later.

**Comment by SpicyAlfredo** [comment]
Maybe a bigger game impact, they feel like late game filler then stuff that effects game play

**Comment by Greatexperiment** [comment]
Yeah, true. Need specific ideas though. :)

**Comment by eliasolsson** [comment]
I honestly think that Expanded Equipment should just be merged into all other tabs - which keeps things consistent. The only issue I can see with this is of course if Paradox were to heavily change something in the research tab. For the Special Research Division, I rarely use it when I'm playing. Don't really know why. I'd agree with them having a bigger game impact though for sure.

**Comment by Greatexperiment** [comment]
I agree with this. We should take the equipment and put in with the other techs - then create a Superweapons/projects tab to replace it. I'll make an issue for that.
